snap/snapcraft.yaml

```diff
@@ -11,7 +11,7 @@ confinement: strict
 parts:
   irccloud-desktop:
     plugin: dump
-    source: https://github.com/irccloud/irccloud-desktop/releases/download/v0.3.0/irccloud-desktop-0.3.0-amd64.deb
+    source: https://github.com/irccloud/irccloud-desktop/releases/download/v0.4.0/irccloud-desktop_0.4.0_amd64.deb
```

Comment:
Like I pointed out in #2 this package will not work with snapd as it currently stands. The code in the deb tries to chown files which is currently disallowed.

Reply:
Oh sorry I didn't see that issue :/ Yeah don't merge this lol.

Reply:
Is there a forum thread for the issue with chowning or is it expected that we modify the program (given snapd tries to avoid requiring that)?

Reply:
I thought there was but I can't find it now